### ISSUE TITLE
Maui Picker with a Title property has no accessible name for screen readers using the UI Automation API

### ISSUE THREAD

**Issue by gbarkerz** [dotnet/maui#6928]
### Description

I created a new MAUI app in Visual Studio Community 2022 V 17.2.0 Preview 6.0, and Microsoft.Maui.Dependencies/Extensions 6.0.300-rc.2.5513. I added a Picker based on demo code at https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker.

When I run the app, Accessibility Insights for Windows reports that the UI Automation combo box element that represents the Picker, has no UI Automation Name property set. Given that the Picker has a Title property set in XAML, perhaps that Title might be repurposed as the UI Automation Name property if no other deliberate action has been taken to set the Name property?

As things are, when the Narrator screen reader navigates to the sample Picker, all it says is "combo box, collapsed", and does not announce the purpose of the ComboBox.

The image below shows Accessibility Insights for Windows reporting the UI Automation ControlType and Name for the Picker. All it shows is "combo box".

![MauiPicker](https://user-images.githubusercontent.com/77085891/167283642-0da1f792-4767-4880-83e3-5d93ca7af7e4.jpg)

### Public API Changes

No public API changes would be required to support this change. 

### Intended Use-Case

In practice, it seems likely that many Maui apps will ship without the product designers giving full consideration of all the ways in which users interact with their devices. For example, customers who use screen readers or who use speech input to control the UI. So if a Picker control ships without use of semantic properties or AutomationProperties, currently a Picker control has no accessible name through the UI Automation API, and when the Narrator screen reader encounters the control, Narrator does not inform the customer of the purpose of the control. 

If the Picker has a Title set, (as it does in sample code at https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker, "Select a monkey"), and the UI Automation Name property is set from that Title when semantic properties or AutomationProperties have not been used to set the Name property, then Narrator and other assistive technologies could leverage that Name and deliver a more usable experience for the customer.
